packages/dataviews/src/types.ts

```diff
@@ -332,6 +332,11 @@ export interface ViewGrid extends ViewBase {
 		 * The fields to use as badge fields.
 		 */
 		badgeFields?: string[];
+
+		/**
+		 * True if the grid has a 1:1 aspect ratio the default, and false if it doesn't.
+		 */
+		hasOneOneAspectRatio?: boolean;
```

Comment:
Should we instead have the possibility to actually pick an aspect ratio (like an enum instead of a boolean)?